app/models/tag.rb

```diff
@@ -1276,6 +1277,23 @@ def suggested_parent_ids(parent_type)
                                        flatten.compact.uniq
   end
 
+  # Get all tags that have this one as a suggested parent.
+  def suggested_child_tags_query(options = {})
+    TagQuery.new(options.merge(
+      unwrangleable: false,
+      unwrangled: false,
+      "pre_#{self.type.downcase}_ids": [self.id]
+    ))
+  end
+
+  def suggested_child_tags(options = {})
+    suggested_child_tags_query(options).search_results
+  end
+
+  def suggested_child_ids
+    suggested_child_tags.pluck(:id)
```

Comment:
It's probably faster not to go through the database for this, because ES will return the IDs already. I think you might be able to extract them with some code like
```
suggested_child_tags_query(options).search.dig("hits", "hits").map { |hit| hit["id"] }
```
But it can also be left as-is if you don't mind having this step load the tags. (Note that it loads the complete tags, not just the IDs -- `pluck` is basically just acting like `map` here because `load_from_elasticsearch` converts the results to an array.)

Reply:
It might be worth adding:
```
# Returns the IDs without loading all items
def ids
    hits.map { |item| item['_id'] }
end
```
to query_result.rb, so we can do:
```
suggested_child_tags_query.search_results.ids
```
and keep the ES-specific structures nicely wrapped.

Reply:
We're getting child tags from the database now, this section has been removed.